resources/views/invoices/edit.blade.php

```diff
@@ -1725,6 +1729,14 @@ function deleteDocument(file) {
 		refreshPDF(true);
 	}
 
+    function showPermissionErrorModal() {
+        swal({!! json_encode(trans('texts.please_register')) !!});
```

Comment:
It looks like it may show multiple errors

Reply:
Implementation did not get brought over in the PR, need to fix.